lib/RESTClient.js

```diff
@@ -393,6 +395,53 @@ class RESTClient {
             callback(errors.InternalError);
         }).end();
     }
+
+    /**
+    *   send a request to get all raft sessions
+    *   get server's response and return it
+    *
+    *   @param {string} reqUids - the identifier of the request
+    *   @param {callback} callback - callback(err, info)
```

Comment:
minor: I don't know how to document this in JSDoc, but it'd be nice to document the format a bit here.